cmake/KokkosCore_config.h.in

```diff
@@ -62,6 +62,7 @@
 #cmakedefine KOKKOS_ENABLE_LIBDL
 #cmakedefine KOKKOS_ENABLE_LIBQUADMATH
 #cmakedefine KOKKOS_ENABLE_ONEDPL
+#cmakedefine KOKKOS_ENABLE_ROCTHRUST
```

Comment:
You might want to add that setting to `HIP::print_configuration()`

@masterleinad same comment about `KOKKOS_ENABLE_ONEDPL` and `SYCL`

Reply:
Let me do that after #6758 is merged.